spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveViews.scala

```diff
@@ -53,6 +57,17 @@ case class ResolveViews(spark: SparkSession) extends Rule[LogicalPlan] with Look
       loadView(catalog, ident)
         .map(createViewRelation(parts, _))
         .getOrElse(u)
+
+    case u@UnresolvedTableOrView(CatalogAndIdentifier(catalog, ident), _, _) =>
+      loadView(catalog, ident)
+        .map(_ => ResolvedV2View(catalog.asViewCatalog, ident))
+        .getOrElse(u)
+
+    case RenameTable(ResolvedV2View(oldCatalog, oldIdent), CatalogAndIdentifier(newCatalog, newIdent), true) =>
```

Comment:
What is `true`? Can we add a name to this so that it is more readable?

Reply:
Is it possible for the rule on line 61 to resolve the multi-part `to` identifier before this rule runs?

Reply:
I added this to `isView@true` so that it's clearer. I checked the Spark code and the `to` identifier isn't resolved anywhere and just passed as-is when doing a rename